themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
>I did not think the change was a drastic one.

It isn't, but it's visible and different to all existing versions. ... we are dealing with the "base" layout here, which has great impact for everyone.